synapse/handlers/federation.py

```diff
@@ -943,9 +943,18 @@ async def backfill(self, dest, room_id, limit, extremities):
 
         return events
 
-    async def maybe_backfill(self, room_id, current_depth):
+    async def maybe_backfill(self, room_id: str, current_depth: int, limit: int) -> bool:
```

Comment:
So...this seems to return a `bool` in a couple of places, but not everywhere. But the caller doesn't care about the return value?

We can either:

1. Not add a return value type hint.
2. Fix line 963 to be `return True`.
3. Change the  *other* returns to just bare `return` statements (with no value).

Reply:
> So...this seems to return a `bool` in a couple of places, but not everywhere. But the caller doesn't care about the return value?
> 
> We can either:
> 
> 1. Not add a return value type hint.
> 2. Fix line 963 to be `return True`.
> 3. Change the  _other_ returns to just bare `return` statements (with no value).

I just did 2.